### DISCUSSION TITLE
DNLA port range

### DISCUSSION THREAD

**Discussion by MarcelCoding** [jellyfin/jellyfin#5874] [Q&A]
Does DNLA (the jellyfin dlna implementation) has a static port range where ports with be used to communicate with devices? If yes, it would be nice to know it. This would made the use with firewalls more easy. If not it would be nice to add a config value to control this behavior. 
In addition it would also be nice to be able to select a subnet witch jellyfin should scan for dlna capable devices.

**Comment by PrplHaz4** [comment]
I don't believe there is a default range, but it is configurable - @BaronGreenback do you know if the docs were ever updated with the new functionality/config options for DLNA/Networking?

**Reply by BaronGreenback** [reply]
The new UDP subsystem didn't make the 10.7 release. 

It's one of the many additions to new plugin version which I'm currently testing at the moment.(https://github.com/BaronGreenback)

**Reply by MarcelCoding** [reply]
@BaronGreenback do you have you plugins in some plugin repository?

**Reply by MarcelCoding** [reply]
Or do I have to create my own local?

**Reply by BaronGreenback** [reply]
There is not repo yet. Once testing is done, the plan is to move it across to JF.

It's very much in alpha testing at the moment - works with build https://github.com/jellyfin/jellyfin/pull/4543 which is based upon the current unstable + some other PR's. You're welcome to try it - but be warned, it will contain bugs.

**Reply by MarcelCoding** [reply]
Ok thank you

**Reply by MarcelCoding** [reply]
@BaronGreenback Do you also plan the second feature what I mentioned? (configuration of the subnet witch should be scanned)

**Reply by BaronGreenback** [reply]
DLNA works via multicast - which is sent out of the internal interfaces.  Requests arriving on a non LAN defined interface are ignored.

If this is insufficient, the only way in which this could be done is outside of JF.

**Reply by MarcelCoding** [reply]
Ok
